cardano-cli/test/cli/genesis-create/data/genesis.spec.json

```diff
@@ -1,19 +1,42 @@
 {
-    "DecentralisationParam": 0.99,
-    "ActiveSlotsCoeff": 0.99,
-    "ProtocolMagicId": 838299499,
-    "StartTime": "2020-01-01T00:20:40Z",
-    "GenDelegs": {},
-    "UpdateQuorum": 12,
-    "MaxHeaderSize": 8192,
-    "MaxMajorPV": 25446,
-    "MaxBodySize": 2097152,
-    "MaxLovelaceSupply": 100000000,
-    "InitialFunds": {},
-    "NetworkMagic": 4036000900,
-    "EpochLength": 21600,
-    "SecurityParam": 2160,
-    "SlotLength": 20,
-    "SlotsPerKESPeriod": 216000,
-    "MaxKESEvolutions": 1080000
+    "activeSlotsCoeff": 0.99,
+    "protocolMagicId": 838299499,
+    "startTime": "2020-01-01T00:20:40Z",
+    "genDelegs": {},
+    "updateQuorum": 12,
+    "maxMajorPV": 25446,
+    "maxLovelaceSupply": 100000000,
+    "initialFunds": {},
+    "networkMagic": 4036000900,
+    "epochLength": 21600,
+    "securityParam": 2160,
+    "slotLength": 20,
+    "slotsPerKESPeriod": 216000,
+    "maxKESEvolutions": 1080000,
+    "protocolParams": {
+        "a0": 0,
+        "decentralisationParam": 0.99,
+        "eMax": 0,
+        "extraEntropy": {
+            "tag": "NeutralNonce"
+        },
```

Comment:
@erikd @dcoutts: Any suggestions as to how this value should be formatted? Right now, I'm just deriving the JSON instances for this type, `Nonce`.

Reply:
Morally it's a Maybe, and a leaf one at that. So it could easily be empty/null or a hash string (as hex, which is probably already the default for that).